src/node_crypto.cc

```diff
@@ -120,6 +120,8 @@ const char* const root_certs[] = {
 #include "node_root_certs.h"  // NOLINT(build/include_order)
 };
 
+const char* extra_root_certs_file;
```

Comment:
done, note that `root_certs` a few lines above is not static, but appears that it could be (I was mirroring its definition). `root_cert_store` below is explicitly declared extern, it shouldn't be static.